src/libraries/System.Runtime/tests/System.Reflection.Tests/FieldInfoTests.cs

```diff
@@ -121,6 +123,29 @@ public void GetValueWithFunctionPointers(Type type, string name, object obj, obj
             Assert.Equal(expected, fieldInfo.GetValue(obj));
         }
 
+        [Fact]
+        public void GetValueFromRvaField()
+        {
+            byte[] valueArray = new byte[] { 1, 2, 3, 4, 5 };
+
+            // Roslyn uses SHA256 of raw data as data field name
```

Comment:
Interesting; this is likely why the original RVA tests added with the fast field feature didn't fail.